### PR TITLE
Create Stripe Form

### LINKED ISSUE #60: [React] Create stripe form
In this task we need to use react-stripe-js to create stripe form in the front-end

### PR DESCRIPTION
**Pull request by tariqabusada**
Create stripe form component

### CODE REVIEW

**Review by abdou059 — APPROVED** [review]

**Review by mai-mohammed — APPROVED** [review]

**Review by husamkamal — APPROVED** [review]

File: client/package.json [hunk 1]
```diff
@@ -21,6 +23,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
+    "react-stripe-checkout": "^2.6.3",
```
Comment (LinaYahya) [inline client/package.json]:
```suggestion
    "react-stripe-checkout": "^2.6.3",
```
uninstall this package as we don't use it

File: client/src/components/HeroSection/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ function HeroSection() {
       />
       <CssBaseline />
       <Container
-        className="hero-container"
+        className="container"
```
Comment (LinaYahya) [inline client/src/components/HeroSection/index.tsx:24]: change to `hero-container`

File: client/src/components/HeroSection/style.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
   position: absolute;
 }
 
-.hero-container{
+.container{
```
Comment (LinaYahya) [inline client/src/components/HeroSection/style.css:8]:
```suggestion
.hero-container{
```

File: client/src/index.css [hunk 4]
```diff
@@ -24,7 +24,4 @@
   /*   min-height : 100vh;
  */
 }
-body{
```
Comment (LinaYahya) [inline client/src/index.css:27]: I guess we need these changes

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — COMMENTED** [review]
Also style the modal form

**Pull request merged**